documentation/markup.rst

```diff
@@ -246,14 +246,16 @@ directives.)  Looking at this example,
 
 ::
 
-   .. function:: foo(x)
-                 foo(y, z)
+   .. function:: remove(path, *, dir_fd=None)
+                 unlink(path, *, dir_fd=None)
       :bar: no
 
-      Return a line of text input from the user.
+      Remove (delete) the file *path*.
 
 ``function`` is the directive name.  It is given two arguments here, the
 remainder of the first line and the second line, as well as one option ``bar``
+By providing multiple arguments, the definitions will be grouped instead of
+separated by a empty line.
```

Comment:
Here you technically provided multiple *names* with the same *arguments*.  The original example had the same *name* (twice) and multiple different *arguments*.
I'm not sure if there is a term that applies to both the *name* and the *argument* -- maybe *lines* is ok:
```suggestion
By providing multiple lines, the definitions will be grouped instead of
separated by a empty line.
```